index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio | Edwin Gichuhi</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <a href="#">My Logo</a>
+                <span><img src="./assets/icons/Union.png" alt="menu"></span>
+            </nav>
+        </header>
+        <section class="headline-section">
+            <p class="intro-text">I'm Edwin, Glad to see you!</p>
+            <p class="details-text">I’m a software developer! I can help you build a product, feature or website.
+                Look through some of my work and experience! If you like what you see and have
+                a project you need coded, don’t hesitate to contact me.
+            </p>
+            <p class="socials-header">LET'S CONNECT</p>
+            <span class="social-media">
+                <a href="http://twitter.com"><img src="./assets/icons/Twitter icon.svg" alt="twitter"></a>
+                <a href="http://linkedin.com"><img src="./assets/icons/Linkedin icon.svg" alt="linkedin"></a>
+                <a href="http://facebook.com"><img src="./assets/icons/Meta icon.svg" alt="meta"></a>
+                <a href="http://github.com"><img src="./assets/icons/Github icon.svg" alt="github"></a>
+                <a href="http://vector.com"><img src="./assets/icons/Vector icon.svg" alt="vector"></a>
+            </span>
```

Comment:
- [ ] I suggest wrapping the social media icons inside the `<ul>` tag and each one in a `<li>` and keep the `<a>` tag that will make your social media icons as a list which it's more easy to read as a code  and also  improve the SEO benchmarks

Reply:
Fixed.